modules/tinymce/src/themes/silver/test/ts/webdriver/editor/TabbingTest.ts

```diff
@@ -0,0 +1,24 @@
+import { Assertions, RealKeys } from '@ephox/agar';
+import { describe, it } from '@ephox/bedrock-client';
+import { TinyHooks } from '@ephox/mcagar';
+import { Focus, Insert, SugarElement } from '@ephox/sugar';
+
+import Editor from 'tinymce/core/api/Editor';
+import Theme from 'tinymce/themes/silver/Theme';
+
+describe('webdriver.tinymce.themes.silver.editor.TabbingTest', () => {
+  const hook = TinyHooks.bddSetup<Editor>({
+    base_url: '/project/tinymce/js/tinymce',
+  }, [ Theme ]);
+
+  it('TINY-3707: Should focus on text editor when tabbing into it', async () => {
+    const editor = hook.editor();
+    const textInput = SugarElement.fromHtml('<input>') as SugarElement<HTMLElement>;
+    const editorElement = SugarElement.fromDom(editor.getElement());
+    Insert.before(editorElement, textInput);
```

Comment:
This `textInput` element also needs to be removed from the dom when the test completes.

- [ ] Ensure input element is cleaned up

Reply:
Just curious, why the element needs to be removed?

Reply:
Because otherwise it'll stay on the page indefinitely, which could cause other tests to fail.